.github/pull_request_template.md

```diff
@@ -0,0 +1,41 @@
+# Background
```

Comment:
Can this be done as a separate PR... I think there's discussion we want to have around it first.

Reply:
Sure, we can leave that to another PR.

Reply:
Ah woops - just a heads up on this - I intended to provide a barebones template. Its updated now to that, but I will strip this out into a separate PR momentarily. 👍